### PR TITLE
refactor of the bdd specs (cypress/FE-1738)

### PR DESCRIPTION
**Pull request by DlgSHi**
refactor of the bdd specs

### CODE REVIEW

File: cypress/features/regression/inlineInputs.feature [hunk 1]
```diff
@@ -31,14 +31,11 @@ Feature: InlineInputs component
       | <>                       |
 
   @positive
-  Scenario Outline: Change second inline input to <label>
-    When I set second inline input to "<label>"
-    Then second inline input on preview is "<label>"
+  Scenario Outline: Change second inline input to <input>
+    When I set second inline input to "<input>"
```
Comment (marcinstachowicz) [inline cypress/features/regression/inlineInputs.feature]: improvement suggestion:
Steps:
```
 Scenario Outline: Change second inline input to <label>
    When I set 2nd inline input to "<label>"
    Then 2nd inline input on preview is "<label>"
```
Step defs:
```
When('I set {int}{word} inline input to {string}', (number, word, text) => {
  inlineInputs(number).clear().type(text);
  pressTABKey(); //to release focus
});

Then('{int}{word} inline input on preview is {string}', (number, word, text) => {
  inlineInputs(number).should('have.value', text);
});
```
Locators:
```
export const inlineInputs = index => cy.iFrame(INLINE_INPUTS)
  .eq(`${index - 1}`);

export const INLINE_INPUTS = 'input';
```
hint: locator `export const INLINE_INPUTS = 'input[data-element="input"]';` not working for me so I suggest to changed it
hint2: adjust input data per scenario

File: cypress/features/regression/link.feature [hunk 2]
```diff
@@ -50,48 +50,36 @@ Feature: Link component
...
-      | tooltipAlign |
-      | left         |
-      | right        |
-      | top          |
-      | bottom       |
-      | center       |
 
   @positive
-  Scenario Outline: Change tooltip position to <tooltipPosition>
+  Scenario Outline: Change tooltip align to <tooltipAlign> and tooltipPosition to <tooltipPosition>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: We could, but we have mutually exclusive cases. Bottom top / left and right. So

[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/features/regression/link.feature]: understood

File: cypress/features/regression/link.feature [hunk 3]
```diff
@@ -50,48 +50,36 @@ Feature: Link component
...
+      | bottom       | top             |
+      | center       | right           |
 
   @positive
   Scenario: Check tabbable and focus the link componenent
-    When I check tabbable checkbox
+    When I uncheck tabbable checkbox
+      And I check tabbable checkbox
     Then Link is tabbable
       And I hit Tab key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: fixed

File: cypress/features/regression/inlineInputs.feature [hunk 4]
```diff
@@ -31,14 +31,11 @@ Feature: InlineInputs component
       | <>                       |
 
   @positive
-  Scenario Outline: Change second inline input to <label>
-    When I set second inline input to "<label>"
-    Then second inline input on preview is "<label>"
+  Scenario Outline: Change second inline input to <input>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/inlineInputs.feature]: agree

File: cypress/features/regression/inlineInputs.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/inlineInputs.feature]: fixed

**Pull request merged**